frontend/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM node:13.12.0-alpine
 
 # set working directory
-WORKDIR /app
+WORKDIR /home/app
```

Comment:
Replace the file like so and this should fix your docker-compose issue with the frontend.
```suggestion
# Start with a base image containing Java runtime
FROM openjdk:8-jdk-alpine

# Add Maintainer Info
LABEL maintainer="sladynnunes98@gmail.com"

# Add a volume pointing to /tmp
VOLUME /tmp

# Make port 8080 available to the world outside this container
EXPOSE 8080

# The application's jar file
ARG JAR_FILE=target/custom-distribution-service-0.0.1.jar

# Add the application's jar to the container
ADD ${JAR_FILE} custom-distribution-service.jar

# Run the jar file 
ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/custom-distribution-service.jar"]
```